examples/demo/src/index.ts

```diff
@@ -38,6 +38,7 @@ const tailwindConfig: TailwindConfig = {
       },
     },
   },
+  plugins: ['@tailwindcss/typography', '@tailwindcss/line-clamp'],
```

Comment:
I like this idea! Maybe we can allow users to build their own worker, so they can implement any plugins they want, and we can avoid including any plugins at all.

`custom-tailwind.worker.ts`

```ts
import lineClamp from '@tailwindcss/line-clamp';
import typography from '@tailwindcss/typography';
import { addPlugin } from 'monaco-tailwindcss/tailwindcss.worker.js'

registerPlugins(lineClamp, typography)
```

Then in the main thread:

```ts
window.MonacoEnvironment = {
  getWorker(moduleId, label) {
    switch (label) {
      case 'tailwindcss':
        return new Worker(new URL('custom-tailwindcss.worker.js', import.meta.url));
      // …
    }
  }
}

configureMonacoTailwindcss({
  tailwindConfig: {
    plugins: ['foo', 'bar']
  }
})
```

Or maybe we don’t need the plugins array in `tailwindConfig` at all.

Reply:
that seems like a ways cleaner and more extensible way.

I the above worker the way you imagine it the api? this seems quite side effecty to me...

Reply:
https://github.com/tailwindlabs/tailwindcss-line-clamp/blob/a1f31f0ae4a5f219e8fcb18697b33818c3a1a421/src/index.js#L1

but one needs to install tailwind then additionally to allow the plugin function... Maybe we can export it? Otherwise the build could be very inefficient right?